steps/incident-update/step.yaml

```diff
@@ -0,0 +1,54 @@
+# Step metadata example - full description at:
+# https://github.com/puppetlabs/relay-rfcs/blob/master/content/0006-integration-layout/rfc.md#common-metadata
+apiVersion: integration/v1
+kind: Step
+name: incident-update
+version: 1
+summary: This step updates a VictorOps timeline from a Relay workflow.
+
+description: |
+  Use this step to feed information back into VictorOps. It will update the
+  account timeline and optionally, if an `entity_id` for an existing incident
+  is supplied to the step, will associate the update with that incident. It
+  posts as `INFO` level messages. It requires that you enable the REST
+  integration on your VictorOps account; doing so will generate a unique URL
+  that you should store as a Secret named `endpoint_url` in Relay, associated with this workflow.
+
+publish:
+  repository: relaysh/victorops-step-incident-update
+
+# optional list of examples; each one can be a Step or a complete Workflow
+examples:
+- summary: Example usage for the step
+  apiVersion: v1
+  kind: Workflow
```

Comment:
It seems like this example would make a lot more sense as a `kind: Step` instead?